server/src/main/java/org/elasticsearch/cluster/metadata/MetaData.java

```diff
@@ -1270,28 +1270,26 @@ public MetaData build() {
             return aliasAndIndexLookup;
         }
 
-        public static String toXContent(MetaData metaData) throws IOException {
-            XContentBuilder builder = XContentFactory.contentBuilder(XContentType.JSON);
-            builder.startObject();
-            toXContent(metaData, builder, ToXContent.EMPTY_PARAMS);
-            builder.endObject();
-            return Strings.toString(builder);
-        }
-
         public static void toXContent(MetaData metaData, XContentBuilder builder, ToXContent.Params params) throws IOException {
             XContentContext context = XContentContext.valueOf(params.param(CONTEXT_MODE_PARAM, "API"));
 
-            builder.startObject("meta-data");
+            if (context == XContentContext.API) {
+                builder.startObject("metadata");
+            }
+            else {
+                builder.startObject("meta-data");
+                builder.field("version", metaData.version());
+            }
 
-            builder.field("version", metaData.version());
             builder.field("cluster_uuid", metaData.clusterUUID);
+            // This is required for some serialization under API context, as some tests in MetaDataTests assert its presenc
```

Comment:
I'm not sure I understand this comment? Is it outdated?

Reply:
Yes sorry. This was added in between the commits, as was meant to say that `ClusterState.toXContent` will now start to emit `cluster_uuid_committed` in its serialization, as I retried to remove `cluster_uuid_committed` before to keep `ClusterState.toXContent` serialization unchanged, but had to put back due to the test asserting it.